tracer/src/Datadog.Trace/Iast/SensitiveData/EvidenceRedactor.cs

```diff
@@ -19,20 +19,12 @@ namespace Datadog.Trace.Iast.SensitiveData;
 
 internal class EvidenceRedactor
 {
+    private static bool _timeoutExceptionErrorShown = true;
```

Comment:
In both places it's set, you ` _timeoutExceptionErrorShown = true`. Given the variable name, I think it should default to `false` here.

However, the use of the past tense makes this slightly confusing. I would call it something like this:
```suggestion
    private static bool _showTimeoutExceptionError = true;
```

Reply:
Yes, you're right. Thanks for spotting this.